### ISSUE TITLE
File Explorer: Copy instead of move between nodes

### ISSUE THREAD

**Issue by thisisparker** [tailscale-dev/vscode-tailscale#201]
I'm not sure if this is a common expectation but it's mine: for dragging files between nodes, I expect it to copy and not move the file. (Within a node, I do expect a move.) A common use case for me, for example, is cloning an unconfigured git repo somewhere and copying an existing config file over, where I wouldn't want it deleted from the source location.

Even if not changing the default behavior, it'd be nice to be able to cut or copy while highlighting a file. If I'm connected to very lived-in computers, there are lots of directories to scroll past.

**Comment by samlinville** [comment]
AC:
- Dragging a file between nodes copies the file to the target node, and does not remove it from the originating node.

I'd like to support a copy/paste keyboard shortcut/context menu workflow, but there are some edge cases I want to think through before committing to that. Will create a separate issue for that work.

**Comment by tylersmalley** [comment]
I agree we should change this behavior to copy instead of move. There is [a lot of checking](https://github.com/tailscale-dev/vscode-tailscale/blob/main/src/node-explorer-provider.ts#L324-L332) we do verify the copy was successful before deleting it from the source. We should move this to a configuration option, something like `tailscale.fileExplorer.moveFileBetweenNodes`
